cmd/zt_sync_local_blob_windows_test.go

```diff
@@ -40,6 +41,8 @@ func (s *cmdIntegrationSuite) TestSyncUploadWithExcludeAttrFlag(c *chk.C) {
 	excludeAttrsStr := "H;I;S"
 	scenarioHelper{}.setAttributesForLocalFiles(c, srcDirName, filesToExclude, attrList)
 
+	time.Sleep(5 * time.Second)
```

Comment:
why the sleep?

Reply:
I'm honestly not sure why, but it made this test less flaky. Perhaps I was going nuts.